### PR TITLE
Fixing typos and clarifying descriptions.

### PR DESCRIPTION
**Pull request by dogmaphobic**

### CODE REVIEW

File: message_definitions/v1.0/common.xml [hunk 1]
```diff
@@ -2499,27 +2499,27 @@
       </entry>
     </enum>
     <enum name="MAV_BATTERY_CHARGE_STATE">
-      <description>Enumeration for states of low battery extent</description>
+      <description>Enumeration for states of low battery</description>
```
Comment (hamishwillee) [inline message_definitions/v1.0/common.xml]: Enumeration for low battery states (?)

File: message_definitions/v1.0/common.xml [hunk 2]
```diff
@@ -2499,27 +2499,27 @@
...
       <entry value="2" name="MAV_BATTERY_CHARGE_STATE_LOW">
         <description>Battery state is low, warn and monitor close</description>
       </entry>
       <entry value="3" name="MAV_BATTERY_CHARGE_STATE_CRITICAL">
-        <description>Battry state is critical, return / abort immediately</description>
+        <description>Battery state is critical, return / abort immediately</description>
       </entry>
       <entry value="4" name="MAV_BATTERY_CHARGE_STATE_EMERGENCY">
-        <description>Battry state is too low for ordinary abortion, fastest possible emergency stop preventing damage</description>
+        <description>Battery state is too low for ordinary abortion, fastest possible emergency stop preventing damage</description>
```
[the remark this answers is not in the document]
Comment (dogmaphobic) [reply message_definitions/v1.0/common.xml]: Yes, it didn't sound good to me either but I was primarily concerned with the "Battry" typo and didn't pay a great deal of attention to the strings themselves.

[the remark this answers is not in the document]
Comment (dogmaphobic) [reply message_definitions/v1.0/common.xml]: I also don't like the lack of punctuation throughout....

[the remark this answers is not in the document]
Comment (hamishwillee) [reply message_definitions/v1.0/common.xml]: > I also don't like the lack of punctuation throughout....

YES! I was going to say that. I might go through in one go and add full stops everywhere. Not a priority.

File: message_definitions/v1.0/common.xml [hunk 3]
```diff
@@ -2499,27 +2499,27 @@
...
       <entry value="2" name="MAV_BATTERY_CHARGE_STATE_LOW">
         <description>Battery state is low, warn and monitor close</description>
       </entry>
       <entry value="3" name="MAV_BATTERY_CHARGE_STATE_CRITICAL">
-        <description>Battry state is critical, return / abort immediately</description>
+        <description>Battery state is critical, return / abort immediately</description>
       </entry>
       <entry value="4" name="MAV_BATTERY_CHARGE_STATE_EMERGENCY">
-        <description>Battry state is too low for ordinary abortion, fastest possible emergency stop preventing damage</description>
+        <description>Battery state is too low for ordinary abort sequence. Perform fastest possible emergency stop to damage</description>
```
[the remark this answers is not in the document]
Comment (dogmaphobic) [reply message_definitions/v1.0/common.xml]: Yup, one missing word and it changes the whole thing, doesn't it? :)

[the remark this answers is not in the document]
Comment (dogmaphobic) [reply message_definitions/v1.0/common.xml]: Done

**Comment by amilcarlucas** [comment]
Squashed and merged, thanks.

**Pull request merged**